src/commands/org/open.ts

```diff
@@ -98,6 +97,7 @@ const generateFileUrl = async (file: string, org: Org): Promise<string> => {
     ) {
       throw error;
     }
-    return '';
+    // fall back to generic frontdoor URL
+    return org.getFrontDoorUrl();
```

Comment:
see the related updated unit tests, if failed to resolve `--source-file` we still want. to open the org.